### PR TITLE
Remove prisma deps from worker

### LINKED ISSUE #2227: Remove prisma deps from worker
1) Separate task creation and pulling from main worker script
2) Move worker code into Util function in bricks
3) Use it on services instead of copying

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #2227 

## PR includes

- [x] Refactor

### CODE REVIEW

**Review by awinogradov — CHANGES_REQUESTED** [review]
image

File: src/utils/worker/index.ts [hunk 1]
```diff
@@ -14,154 +13,33 @@ const log = (...rest: unknown[]) => console.log('[WORKER]:', ...rest);
...
-};
+const onRetryLimitExeed = (error: any, job: Job) =>
+    Sentry.captureException(error, {
+        fingerprint: ['worker', 'resolve', 'retry'],
+        extra: {
+            job,
+        },
+    });
+
+const onQueeTooLong = () => Sentry.captureMessage('Queue too long. Smth went wrong.');
```
Comment (LamaEats) [inline src/utils/worker/index.ts:24]: `onQueeTooLong` => `onQueueTooLong`

File: src/utils/worker/index.ts [hunk 2]
```diff
@@ -14,154 +13,33 @@ const log = (...rest: unknown[]) => console.log('[WORKER]:', ...rest);
...
+
+const onError = (error: any) => console.log(error.message);
```
Comment (LamaEats) [inline src/utils/worker/index.ts:26]: Maybe `Sentry.captureException` instead of `console.log`

File: tsconfig.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 {
+    "ts-node": {
+        // these options are overrides used only by ts-node
+        // same as the --compilerOptions flag and the TS_NODE_COMPILER_OPTIONS environment variable
+        "compilerOptions": {
+            "module": "node16",
```
Comment (LamaEats) [inline tsconfig.json:6]: why not node@20?

**Review by LamaEats — CHANGES_REQUESTED** [review]

**Pull request closed without merging**